code/game/objects/structures/watercloset.dm

```diff
@@ -100,6 +100,10 @@
 	else if(istype(I, /obj/item/reagent_containers))
 		if (!open)
 			return
+		if(istype(I, /obj/item/reagent_containers/food/snacks/monkeycube))
+			var/obj/item/reagent_containers/food/snacks/monkeycube/cube = I
+			cube.Expand()
```

Comment:
does this return the last fingerprintslast since it's in attackby without a call to the parent prior?

Reply:
Nope. It does not return last fingerprint for any of these actions, which is unlike other similiar structures. I have added add fingerprints to the top, just like other structures have.